### ISSUE TITLE
General Getting Start Guide

### ISSUE THREAD

**Issue by maxaud** [tapquo/Lungo.js#139]
I'm really trying to get behind Lungo, it seems like it will be a great declarative framework for mobile app development.

I'm having some issues incorporating Lungo into a band new Phonegap/Cordova 2.3 project (iOS).

I tried copying the demo code and I can get the app running but there are no icons.

I also feel there is a lot of bloat or things I may not need to be integrating.

Additionally, doesn't seem like the extra asides seem to be working for me.

Can someone give a quick run-through of what to include in the head and foot of the HTML document, and some of the dependent files that I'll need to make a "Hello World" app?

**Comment by otupman** [comment]
fyi I believe phonegap/cordova isn't the intended platform for Lungo, however it does work but your mileage may vary.

My starting point was the simple 3-section demo that comes with the 2.0.0 version downloadable from the Lungo homepage, from which I ripped it apart to get what I wanted. to keep it simpler I stripped out some of the iOS-specific stylesheets as I didn't want them (yet).

Having done that, I moved it's Lungo.init call to the function handling phonegap's deviceready event.

In the head, I have:
    lungo.css
    lungo.icon.css
    lungo.icon.brand.css
    lungo.theme.default.css

Before the end of the body tag I have:
    quo.js
    lungo.js
    my-own-file.js
    cordova.js

and then the app.initialize() call.

Hope that helps! (or at least shows that it's possible)
